.gitignore

```diff
@@ -14,6 +14,7 @@
 
 # production
 /build
+dev/nextjs-project/data/default.json
```

Comment:
Although this is not an issue, we want to replicate a real environment with the `nextjs-project` so it's better to leave the `default.json file` in.

PS: If this becomes annoying with local development we can add update husky config to ignore it.